packages/cozy-harvest-lib/src/datacards/doctypeToDataCard.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * This file list all possible doctypes to datacard possibilities.
+ * Its default export can be used as the value of the doctypeToDataCard
+ * prop of Harvest's <Routes> component (see Readme.md "Data cards" section).
+ *
+ * Beware that if you import this file, you will have to install
+ * all dependencies that are necessary for all types of data card.
+ */
```

Comment:
Can you see a possible solution to avoid this?

Reply:
Yes, an application can make her own variable `const doctypeToDataCard` or just not import this file.